frontend/lib/identification/id_card/card_content.dart

```diff
@@ -46,11 +48,11 @@ class CardContent extends StatelessWidget {
   const CardContent(
       {super.key, required this.cardInfo, this.region, required this.isExpired, required this.isNotYetValid});
 
-  String get _formattedExpirationDate {
+  String _getFormattedExpirationDate(BuildContext context) {
     final expirationDay = cardInfo.hasExpirationDay() ? cardInfo.expirationDay : null;
     return expirationDay != null
         ? DateFormat('dd.MM.yyyy').format(DateTime.fromMillisecondsSinceEpoch(0).add(Duration(days: expirationDay)))
```

Comment:
Would it be possible to use localized date formats? Not necessary for this PR, but maybe you've read sth about this already.

Reply:
Yes, that is possible. See https://docs.flutter.dev/ui/accessibility-and-internationalization/internationalization#messages-with-dates.

I will create an issue :)